apps/web/src/views/Farms/FarmsV3.tsx

```diff
@@ -358,14 +357,15 @@ const Farms: React.FC<React.PropsWithChildren> = ({ children }) => {
           (v3FarmOnly && farm.version === 3) ||
           (v2FarmOnly && farm.version === 2) ||
           (boostedOnly && farm.boosted && farm.version === 3) ||
-          (stableSwapOnly && farm.isStable),
+          (stableSwapOnly && farm.version === 2 && farm.isStable),
       )
 
-      const stakedFilterFarms = chosenFs.filter(
+      const stakedFilterFarms = chosenFs?.filter(
         (farm) =>
+          farm.version === 2 &&
```

Comment:
what about staked v3 farms?

Reply:
Still work, u can check with the preview link

Reply:
hmm it's a bug on prod as well. You can try turn on both farm type filter and staked only filter.
<img width="621" alt="image" src="https://github.com/pancakeswap/pancake-frontend/assets/116779127/b12e43eb-3d9d-4dda-80a2-53e06926b28c">